src/parser.rs

```diff
@@ -3441,6 +3448,9 @@ impl<'a> Parser<'a> {
                     Some('\'') => Ok(Value::SingleQuotedString(w.value)),
                     _ => self.expected("A value?", Token::Word(w))?,
                 },
+                Keyword::NoKeyword if dialect_of!(self is SnowflakeDialect) => {
```

Comment:
@AugustoFKL Here is how I tried. For some reason, this if clause works when using the cli, but when running tests it fails